installer/templates/new/widgets/text/text.scss

```diff
@@ -19,14 +19,20 @@ $moreinfo-color:    rgba(255, 255, 255, 0.7);
 
   .more-info {
     color: $moreinfo-color;
+    font-size: 24px;
   }
 
   .updated-at {
     color: rgba(255, 255, 255, 0.7);
   }
 
 
-  &.large h3 {
+  & h3 {
     font-size: 65px;
   }
 }
+
+.widget-text-custom {
+
+    background-color: #ff0000;
+}
```

Comment:
I think an include from application.scss makes sense. I will create another PR for that and have it include `userstyles.scss`.